tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever_integration.py

```diff
@@ -40,7 +40,7 @@ def cloud_context_and_batch_request_with_simple_dataframe(
     return context, batch_request
 
 
-@pytest.mark.cloud
+# @pytest.mark.cloud
```

Comment:
I think the static analysis is complaining about this one

Reply:
Oh whoops! Good catch, meant to revert.